src/bootstrap/config.rs

```diff
@@ -1000,6 +1031,30 @@ impl Config {
             toml.merge(included_toml);
         }
 
+        for option in flags.options.iter() {
+            fn get_table(option: &str) -> Result<TomlConfig, toml::de::Error> {
+                toml::from_str(&option)
+                    .and_then(|table: toml::Value| TomlConfig::deserialize(table))
+            }
+
+            if let Ok(value) = get_table(option) {
+                toml.merge_replace(value);
```

Comment:
Why is this new `merge_replace` function necessary? Can't you do `value.merge(toml)` instead?

Reply:
The macro-generated part of the `merge` impl only merges if the `self` field is None. We want to be able to override existing values so we can't use this same behaviour

Reply:
Actually, this could be a single function with a `replace` option. That would fit better with detecting duplicates